tests/test_interface.rb

```diff
@@ -1321,7 +1352,7 @@ def test_interface_ipv4_all_interfaces
       validate_interface_shutdown(inttype_h)
       validate_vrf(inttype_h)
       config(*cfg)
-      InterfaceChannelGroup.new(interfaces[1]).channel_group = false
+      interface_ethernet_default(interfaces[1])
```

Comment:
I vaguely remember a problem where 'default interface' does not clear the channel group on some platform. I suggest doing both:

```
      interface_ethernet_default(interfaces[1])
      InterfaceChannelGroup.new(interfaces[1]).channel_group = false
```